common/src/main/resources/Operator.properties

```diff
@@ -295,6 +318,7 @@ WLSEO-0036=Cluster {0} is unavailable: an insufficient number of its servers tha
 WLSEO-0037=Cluster {0} is incomplete for one or more of the following reasons: there are failures detected, \
   there are pending server shutdowns, or not all servers expected to be running are ready \
   and at their target image, auxiliary images, restart version, and introspect version.
+WLSEO-0038=Persistent volume claim error
```

Comment:
No other info about the error?

Reply:
This is similar to other high-level errors, such as the following errors. The actual error message that gets printed is "WLSKO-0239=PersistentVolumeClaim {0} is not in bound state. Status is : {1}." 

WLSEO-0020=Introspection error
WLSEO-0021=Kubernetes Api call error
WLSEO-0022=Server pod error
WLSEO-0023=Replicas too high
WLSEO-0024=Internal error